src/components/LeftSideMenu.vue

```diff
@@ -190,8 +191,10 @@ export default {
 
         },
         _drafts() {
-            let recipes = this.$store.state.contributor.recipes
-            let cookbooks = this.$store.state.contributor.cookbooks
+            //let recipes = this.$store.state.contributor.recipes
+            let recipes = this.$store.state.active_user.recipes
+            //let cookbooks = this.$store.state.contributor.cookbooks
+            let cookbooks = this.$store.state.active_user.cookbooks
```

Comment:
Same question, what happens when the active user's cookbooks is not set? i.e null